tests/quantization/test_quantized_module.py

```diff
@@ -512,3 +512,49 @@ def test_quantized_module_initialization_error():
             ordered_module_output_names=None,  # This makes the combination invalid
             quant_layers_dict={"layer1": (["input1"], "QuantizedOp")},
         )
+
+
+@pytest.mark.parametrize("model_class, input_shape", [pytest.param(FC, (100, 32 * 32 * 3))])
+def test_crt_simulation_using_cp_vl(model_class, input_shape, default_configuration):
+    "Test simulation based on whether the circuit uses CRT encoding."
```

Comment:
same here, maybe put "Tes that using rounding makes circuits non-CRT" or something like that